integration-tests/e2e/benchmark/README.md

```
@@ -0,0 +1,380 @@
+| `queued-gap-ttl` | 60s | TTL for stalled senders missing head nonce |
+
+### Environment variables
+
+| Variable | Default | Description |
+|---|---|---|
+| `E2E_MINITIAD_BIN` | (auto-build) | Path to prebuilt `minitiad` binary |
+| `BENCHMARK_RESULTS_DIR` | `results/` | Output directory for JSON results |
+
+## Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)